test/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ add_executable(read_events read_events.cc)
 target_include_directories(read_events PUBLIC ${CMAKE_SOURCE_DIR}/edm4hep )
 target_link_libraries(read_events edm4hep podio::podio edm4hepDict )
 add_test(NAME read_events COMMAND read_events)
+set_tests_properties(read_events PROPERTIES DEPENDS write_events)
+
+add_executable(test_rdf test_rdf.cc)
```

Comment:
Instead of requiring the component above we can just check that the library exists
IF(TARGET ROOT::ROOTDataFrame)
  add_executable(....
...
ENDIF()
```

Reply:
That's much more elegant indeed!